templates/test/ci/patches/control-plane-image-ci-version.yaml

```diff
@@ -12,4 +12,4 @@ spec:
           publisher: cncf-upstream
           offer: capi
           sku: ubuntu-2204-gen1
-          version: latest
+          version: "128.2.20230918"
```

Comment:
latest version before the last one from:  az vm image list --publisher cncf-upstream --offer capi --sku ubuntu-2204-gen1 -o table --all